istioctl/pkg/writer/ztunnel/configdump/services.go

```diff
@@ -56,16 +59,32 @@ func (c *ConfigWriter) PrintServiceSummary(filter ServiceFilter) error {
 		}
 		return cmp.Compare(a.Hostname, b.Hostname)
 	})
-	fmt.Fprintln(w, "NAMESPACE\tSERVICE NAME\tSERVICE VIP\tWAYPOINT")
+	fmt.Fprintln(w, "NAMESPACE\tSERVICE NAME\tSERVICE VIP\tWAYPOINT\tENDPOINTS")
 
 	for _, svc := range svcs {
 		var ip string
 		if len(svc.Addresses) > 0 {
 			_, ip, _ = strings.Cut(svc.Addresses[0], "/")
 		}
+		allEndpoints := len(svc.Endpoints)
+		healthyEndpoints := 0
+		for _, ep := range svc.Endpoints {
+			w, f := workloadsByUID[ep.WorkloadUID]
+			if !f {
+				fmt.Println("not found!", ep.WorkloadUID)
+				continue
+			}
+			if w.Status != "Healthy" {
+				fmt.Println("not health", w)
```

Comment:
```suggestion
				fmt.Println("not healthy", w)
```

Reply:
oops these were debug logs, should drop entirely